docs-src/references/strongly-typed-errors/cause-bad-update.md

```diff
@@ -0,0 +1,17 @@
+---
+id: cause-bad-update
+title: Cause Bad Update
+description: Explanation for Cause Bad Update error message, and how to fix it.
+sidebar_label: Cause Bad Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+<!--TODO: add link to Workflow Update page when written -->
+
+This error indicates that a [Workflow Execution](/concepts/what-is-a-workflow-execution) could not be updated.
+
+`BadUpdate` can happen when a [Worker](/concepts/what-is-a-worker) generates a [Workflow Task Completed](/references/events#WorkflowTaskCompleted) message with missing fields or an invalid format.
```

Comment:
Reading this, I don't know when/why this might occur, and whether it's related to the new Update feature, and how to fix it.

Reply:
Going with what information I have. This happens when completion happens before an update is received or if the update is invalid

Reply:
It says below that it may indicate usage of an unsupported SDK.
There's not much else we can say.

Reply:
do you mean an SDK that doesn't support Updates? eg TS workflow receiving an Update?

Reply:
It means somehow an SDK sent a workflow task completion that has a broken update response.
That should never happen with any of our supported SDKs.